internal/nrql/command_query.go

```diff
@@ -29,6 +32,19 @@ issue the query against.
 	PreRun:  client.RequireClient,
 	Run: func(cmd *cobra.Command, args []string) {
 		accountID := configAPI.RequireActiveProfileAccountID()
+
+		_, err := os.Stat(query)
+		if err != nil && !errors.Is(err, os.ErrNotExist) {
+			log.Fatal(err)
+		} else {
+			// query points to a file/file descriptor
+			fileBytes, readErr := os.ReadFile(query)
+			if readErr != nil {
+				log.Fatal(readErr)
+			}
+			query = string(fileBytes)
+		}
+
 		result, err := client.NRClient.Nrdb.QueryWithContext(utils.SignalCtx, accountID, nrdb.NRQL(query))
```

Comment:
It'd be nicer if `QueryWithContext` accepted an `io.Reader` instead of a string. But that seems like a more difficult change.